### PR TITLE
add kite_nrf9160

### PR DESCRIPTION
**Pull request by tautologyclub**

### CODE REVIEW

File: boards/arm/kite_nrf9160/kite_nrf9160_common.dts [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+	};
+
+	en_pmod1: pmod1-pwr-ctrl {
+		compatible = "regulator-fixed-sync", "regulator-fixed";
+		status = "okay";
+		regulator-name = "EN_PMOD1";
+
+		enable-gpios = <&gpio0 21 GPIO_ACTIVE_HIGH>;
+		regulator-boot-on;
+	};
```
Comment (tobsan) [inline boards/arm/kite_nrf9160/kite_nrf9160_common.dts:79]: This is superseded by the code in `board.c`, no? Delete if so

**Review by tobsan — CHANGES_REQUESTED** [review]

File: boards/arm/kite_nrf9160/kite_nrf9160_common.dts (hunk 1 above)
Comment (tautologyclub) [reply boards/arm/kite_nrf9160/kite_nrf9160_common.dts:79]: Actually the code in `board.c` is improved slightly here. Should've made that a separate patch set I guess. But now that we're here, feel free to look at `board.c` and see if it's to your liking

File: boards/arm/kite_nrf9160/board.c [hunk 2]
```diff
@@ -0,0 +1,20 @@
+#include <zephyr/init.h>
+#include <zephyr/device.h>
+#include <zephyr/kernel.h>
+#include <zephyr/drivers/gpio.h>
+
+#if !defined(CONFIG_REGULATOR)
+
+static int enable_regulator_at_boot(const struct device *unused)
```
Comment (tobsan) [inline boards/arm/kite_nrf9160/board.c:16]: A comment here explaining that we're not REALLY using a regulator, just the handy bindings, and we don't recommend treating it as a regulator anyway?

File: boards/arm/kite_nrf9160/kite_nrf9160_common.dts (hunk 1 above)
Comment (tobsan) [reply boards/arm/kite_nrf9160/kite_nrf9160_common.dts:79]: Yep, I think that looks good. See separate comment.

**Review by tobsan — APPROVED** [review]

**Pull request merged**